packages/dashboard/src/pages/Personas/components/PostHistory/PostHistoryRow.tsx

```diff
@@ -0,0 +1,77 @@
+import { Stack, Typography } from '@material-ui/core'
+import { memo, ReactNode, useMemo } from 'react'
+import { FileMessageIcon, ITOIcon, MessageIcon, RedPacketIcon, PollIcon } from '@masknet/icons'
+import { getMaskColor } from '@masknet/theme'
+import { Services } from '../../../../API'
+
+const SUPPORT_PLUGIN: Record<string, ReactNode> = {
```

Comment:
Why does the key not use Enum? 🤔

Reply:
I want use it like this: ` SUPPORT_PLUGIN[plugin] ?? <MessageIcon sx={{ width: 48, height: 48 }} />`